.github/workflows/build_and_test.yml

```diff
@@ -169,7 +169,7 @@ jobs:
     uses: ./.github/workflows/build_reusable.yml
     with:
       skipForDocsOnly: 'yes'
-      afterBuild: NEXT_TEST_MODE=dev node run-tests.js --timings -g ${{ matrix.group }}/3 -c ${TEST_CONCURRENCY} --test-pattern '^(development|e2e|unit)/.*\.test\.(js|jsx|ts|tsx)$'
+      afterBuild: NEXT_TEST_MODE=dev node run-tests.js --timings -g ${{ matrix.group }}/3 -c ${TEST_CONCURRENCY} --test-pattern '^(test\/(development|e2e|unit)|packages\/next\/src\/.*|packages\/font\/src\/.*)/.*\.test\.(js|jsx|ts|tsx)$'
```

Comment:
I wonder if we should change `font` to `*` so that it runs tests in every package?

Reply:
I think we'd need something like `'^(test\/(development|e2e|unit)|packages\/(?!.*\/dist\/.*).*\/.*)/.*\.test\.(js|jsx|ts|tsx)$'` to avoid running tests on `packages/next/dist/**` and `packages/font/dist/**` since those files are available at test time, but the regex starts to get a bit ugly. What do you think?